### ISSUE TITLE
Run example code on CPU

### ISSUE THREAD

**Issue by aleksmaksimovic** [SalesforceAIResearch/uni2ts#115]
Hello,

how can I run the code sample shown in the section
 🏃 Getting Started on my CPU device?

When running the code, I get the error 

"RuntimeError: The NVIDIA driver on your system is too old (found version 10020). Please update your GPU driver by downloading and installing a new version from the URL: http://www.nvidia.com/Download/index.aspx Alternatively, go to: https://pytorch.org to install a PyTorch version that has been compiled with your version of the CUDA driver.".

Unfortunately, at the moment I'm not able to update my CUDA driver, so I have to stick to the CPU.

Is there a way to run this code example on the CPU?

**Comment by chenghaoliu89** [comment]
@aleksmaksimovic, yes, the model can run on a CPU. Could you show the code snippet  and the full stacktrace with error msg?
